custom/opm/health_status.py

```diff
@@ -320,3 +356,40 @@ def adult_scale(self):
     @property
     def child_scale(self):
         return self.service_available('vhnd_child_scale_available')
+
+    @property
+    def ifa_stock_available(self):
+        return self.service_available('vhnd_ifa_available')
+
+    @property
+    def ors_stock_available(self):
+        return self.service_available('vhnd_ors_available')
+
+    @property
+    def zinc_stock_available(self):
+        return self.service_available('vhnd_zn_available')
+
+    @property
+    def measles_stock_available(self):
+        return self.service_available('vhnd_measles_vacc_available')
+
+    @property
+    def birth_spacing_bonus(self):
+        return len([c for c in self.all_cases if c.birth_spacing_years])
+
+    @property
+    def nutritional_bonus(self):
+        return len([c for c in self.all_cases if c.weight_grade_normal])
+
+    @property
+    def closed_pregnants(self):
+        return len([c for c in self.all_cases if c.status == 'pregnant' and c.closed_in_reporting_month])
+
+    @property
+    def closed_mothers(self):
+        return len([c for c in self.primary_cases if c.status == 'mother' and c.closed_in_reporting_month])
+
+    @property
+    def closed_children(self):
+        return len([c.num_children for c in self.primary_cases
+                    if c.status == 'mother' and c.closed_in_reporting_month])
```

Comment:
Copying my comment from the old PR:
> This will be the same as closed mothers.  I think what you want is `sum`:
```python
return sum([c.num_children for c in self.primary_cases
            if c.status == 'mother' and c.closed_in_reporting_month])
```

Other than that this is g2m.